semgrep-core/tests/generate_test_matrix.py

```diff
@@ -157,26 +159,28 @@ def generate_cheatsheet(root_dir: str):
                 sgrep_path = find_path(root_dir, lang, category, subcategory, 'sgrep')
                 code_path = find_path(root_dir, lang, category, subcategory, lang_dir_to_ext(lang))
                 
-                higlights = []
+                highlights = []
                 if os.path.exists(sgrep_path) and os.path.exists(code_path):
                     ranges = run_semgrep_on_example(lang, sgrep_path, code_path)
                     if ranges:
                         j = json.loads(ranges)
                         for entry in j['results']:
-                            higlights.append({'start': entry['start'], 'end': entry['end']})
+                            highlights.append({'start': entry['start'], 'end': entry['end']})
 
                 entry = { 
                     'pattern': read_if_exists(sgrep_path),
                     'pattern_path': sgrep_path,
                     'code': read_if_exists(code_path),
                     'code_path': code_path,
-                     'highlights': higlights,
+                    'highlights': highlights,
                 }
                
 
                 #print((lang, entry))
                 feature_name = VERBOSE_FEATURE_NAME.get(category, category)
                 subcategory_name = VERBOSE_SUBCATEGORY_NAME.get(subcategory, subcategory)
+                if (category == "regexp" and subcategory == "string"):
+                    subcategory_name = VERBOSE_REGEXP_SYNTAX
```

Comment:
is there a better way to do this?

Reply:
Only other way I could think of was to re-name the actual test files, because currently this subcategory mapping is based on those names, but that felt kind of hacky too, and I wasn't sure if other things depended on those names